maze.swift

```diff
@@ -1,74 +1,73 @@
 // Maze generator in Swift
-// Joe Wingbermuehle
-// 2014-12-11
+// Joe Wingbermuehle - Sarp Başaraner
+// 2018-02-20
```

Comment:
To make tracking contributions easier, I think it might make sense to switch this format to something like:
```
// 2014-12-11 - Joe Wingbermuehle
// 2018-02-20 - Sarp Başaraner - Update for Swift 4
```

Reply:
Done, please see #4